src/createBannedAttributeRule.js

```diff
@@ -1,114 +1,113 @@
-module.exports = ({ preferred, negatedPreferred, attributes }) => (context) => {
-  function getCorrectFunctionFor(node, negated = false) {
-    return (node.arguments.length === 1 ||
+export default ({ preferred, negatedPreferred, attributes }) => (context) => {
+  const getCorrectFunctionFor = (node, negated = false) =>
+    (node.arguments.length === 1 ||
       node.arguments[1].value === true ||
       node.arguments[1].value === "") &&
-      !negated
+    !negated
       ? preferred
       : negatedPreferred;
-  }
 
   const isBannedArg = (node) =>
     attributes.some((attr) => attr === node.arguments[0].value);
 
   return {
-    [`CallExpression[callee.property.name=/${preferred}|${negatedPreferred}/][callee.object.property.name='not'][callee.object.object.callee.name='expect']`](
+    [`CallExpression[callee.property.name=/${preferred}|${negatedPreferred}/][callee.object.property.name='not'][callee.object.object.callee.name='expect']`]: (
```

Comment:
I actually prefer this syntax over the arrow syntax in this case.  can we revert those please?

Reply:
I’ve done it in order to not violate the max-line linting error so eassily

Reply:
Ok then it probably doesn't make sense for a lint rule repo

Reply:
I think it does, so we only disable it when really necessary

Reply:
hmm alternatively did you try breaking the string into multiple lines?

maybe this works?
```js
[`
CallExpression[callee.property.name=/${preferred}|${negatedPreferred}/]
[callee.object.property.name='not']
[callee.object.object.callee.name='expect']
`](
```

Reply:
@benmonro Breaking the string into multiple lines will make the tests fail

Reply:
gotcha.  ok then we need to disable the rule.  A lint plugin that uses the selector syntax is going to have to disable quite a few of them so it probably makes sense to either change it for the rules directory or something, but I really want to keep this syntax.  it's probably my favorite feature of writing a lint rule. :)

Reply:
We currently only disable the `max-lines-per-function` rule in 1 file

Reply:
alright.  do you mind reverting these back to not use arrow functions then?  In these files I much prefer the object function syntax.

Reply:
I'm on it 😉